ipatests/prci_definitions/gating.yaml

```diff
@@ -30,8 +30,8 @@ jobs:
         git_repo: '{git_repo}'
         git_refspec: '{git_refspec}'
         template: &ci-master-latest
-          name: freeipa/ci-master-f32
-          version: 0.0.16
+          name: freeipa/ci-master-f33
+          version: 0.0.4
```

Comment:
@tiran `0.0.5` is available with updated packages.

Reply:
Perfect, thanks!